guides/common/modules/proc_registering-project-as-a-client-of-keycloak.adoc

```diff
@@ -42,15 +42,16 @@ Use `foreman-openidc` as the application name.
 --keycloak-auth-role root-admin \
 -t openidc -l /users/extlogin --force
 ----
-.. Configure {Project} to use {keycloak} as an authentication source:
+.. Configure {Project} to use {keycloak} as an authentication source for {ProjectWebUI}:
 +
 [options="nowrap", subs="verbatim,quotes,attributes"]
 ----
 # {foreman-installer} --foreman-keycloak true \
 --foreman-keycloak-app-name "foreman-openidc" \
 --foreman-keycloak-realm "_{Project}_Realm_"
 ----
-* If you want users to authenticate by using the Hammer CLI, create a client for {Project}.
+* If you want users to authenticate by using the Hammer CLI:
+.. Create a client for {Project}.
 Use `hammer-openidc` as the application name.
```

Comment:
```suggestion
Use `hammer-openidc` as the application name:
```

line 67 ends with a colon.

Reply:
Because there is another sentence ("Use hammer-openidc [...]") intervening between the lead-in sentence ("If you want [...]") and the list, period is the correct punctuation mark to use. (The IBM Style guide)